.changeset/spicy-walls-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
This would be a `patch` for the nonbreaking change, since the package is on version 0.x.